addons/sourcemod/scripting/ttt/ttt_shop.sp

```diff
@@ -1063,9 +1063,14 @@ public void TTT_OnRoundEnd(int WinningTeam)
 	}
 }
 
-public void TTT_OnBodyFound(int client, int victim, const char[] deadPlayer)
+public void TTT_OnBodyFound(int client, int victim, const char[] deadPlayer, int isBodySilecedID = 0)
 {
-	addCredits(client, g_cCreditsFoundBody.IntValue);
+	bool bSilentID = g_cSilentIdEnabled.BoolValue;
+	bool bSilentRewards = g_cSilentIdRewards.BoolValue;
```

Comment:
g_cSilentIdEnabled and g_cSilentIdRewards are not defined in shop

Reply:
Fixed